include/ck/tensor_operation/gpu/device/device_batched_gemm_reduce_xdl_cshuffle.hpp

```diff
@@ -508,13 +507,11 @@ struct DeviceBatchedGemmReduce_Xdl_CShuffle : public DeviceGemmReduce<AElementwi
         ComputeBasePtrOfStridedBatch(index_t BatchStrideA,
                                      index_t BatchStrideB,
                                      index_t BatchStrideC,
-                                     index_t BatchStrideD0,
-                                     index_t BatchStrideD1)
+                                     index_t BatchStrideD)
             : BatchStrideA_(BatchStrideA),
               BatchStrideB_(BatchStrideB),
               BatchStrideC_(BatchStrideC),
-              BatchStrideD0_(BatchStrideD0),
-              BatchStrideD1_(BatchStrideD1)
+              BatchStrideD_(BatchStrideD)
```

Comment:
Cannot use a single BatchStrideD for all the D tensors, because you cannot assume all the D tensors have the same stride

Reply:
We do not have sequence of strideD in MakeArgument() and have only one d descriptor so far.
I will keep flexibile of multiple strideD in the kernel 20ae1ce93a81f4e9559d4a6fd6ffb15f73510d87